leptos_i18n/Cargo.toml

```diff
@@ -114,62 +114,63 @@ track_locale_files = ["leptos_i18n_macro/track_locale_files"]
 
 
 [package.metadata.cargo-all-features]
+max_combination_size = 1
```

Comment:
I know CI will take some time, but there is a lot of feaures enabling/disabling things and I want to make sure in CI that they don't mess with each other

Reply:
It was debugging leftover!